From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: ColorfulRhino <ColorfulRhino@armbian.com>
Date: Tue, 2 Jul 2024 13:37:16 +0200
Subject: Use semantic versioning for pylibfdt

Setuptools >66 only allows for semantic versioning which old U-Boot
versions don't yet use.
---
 scripts/dtc/pylibfdt/setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/dtc/pylibfdt/setup.py b/scripts/dtc/pylibfdt/setup.py
index 111111111111..222222222222 100755
--- a/scripts/dtc/pylibfdt/setup.py
+++ b/scripts/dtc/pylibfdt/setup.py
@@ -114,7 +114,7 @@ libfdt_module = Extension(
 
 setup(
     name='libfdt',
-    version= version,
+    version='2022.04',
     author='Simon Glass <sjg@chromium.org>',
     description='Python binding for libfdt',
     ext_modules=[libfdt_module],
-- 
Armbian

